templates/CMakeLists.txt.template

```diff
@@ -73,10 +73,11 @@
 
   set(PACKAGE_NAME      "grpc")
   set(PACKAGE_VERSION   "${settings.cpp_version}")
+  set(CORE_VERSION      "${settings.core_version}")
   set(PACKAGE_STRING    "<%text>${PACKAGE_NAME} ${PACKAGE_VERSION}</%text>")
   set(PACKAGE_TARNAME   "<%text>${PACKAGE_NAME}-${PACKAGE_VERSION}</%text>")
   set(PACKAGE_BUGREPORT "https://github.com/grpc/grpc/issues/")
-  project(<%text>${PACKAGE_NAME}</%text> C CXX)
+  project(<%text>${PACKAGE_NAME}</%text> LANGUAGES C CXX)
```

Comment:
nit: is this purely a style improvement?

Reply:
At the moment, yes. IMO we should eventually set `(project ... VERSION)` to be either `PACKAGE_VERSION` or `CORE_VERSION`. At that point we will need to use the longer signature of the `project()` command where LANGUAGES are explicitly separated.

Reply:
Sg, thanks for the explanation.